### PR TITLE
DOCS-3274 Update warning about shell modifications

### PR DESCRIPTION
**Pull request by tonyriv3**
DOCS-3274 Update warning about shell modifications

Grammar, spelling, passive voice, etc.

### CODE REVIEW

File: content/SCALE/SystemSettings/ScaleShell.md [hunk 1]
```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
 weight: 60
 ---
 
-The SCALE shell is a convenient means to run command lines tools, configure different system settings, or find log files or debug information.
-The shell opens with the root user logged in.
+The SCALE **Shell** is convenient for running command lines tools, configuring different system settings, or finding log files and debug information.
+The **Shell** opens with the root user logged in.
+
+{{< hint danger >}}
+**Warning: the supported mechanisms for making configuration changes are the TrueNAS WebUI, CLI, and API exclusively. ALL OTHERS ARE NOT SUPPORTED AND WILL RESULT IN UNDEFINED BEHAVIOR AND MAY RESULT IN SYSTEM FAILURE.**
```
Comment (micjohnson777) [inline content/SCALE/SystemSettings/ScaleShell.md:11]:
```suggestion improper use of bold and all caps. emphasis is already in the hint tags used.
Warning: the supported mechanisms for making configuration changes are the TrueNAS WebUI, CLI, and API exclusively. *All others are not supported and result in undefined behaviors, and might result in system failure.* 
```

File: content/SCALE/SystemSettings/ScaleShell.md [hunk 2]
```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
...
+{{< /hint >}}
 
 ![Shell](/images/SCALE/shell.png "SCALE Shell")
 
-The *Set font size slider* adjusts the size of text displayed in the shell.
-Click *RESTORE DEFAULT* to reset the shell font and size.
+The *Set font size slider* adjusts the **Shell**'s displayed text size.
```
Comment (micjohnson777) [inline content/SCALE/SystemSettings/ScaleShell.md:16]:
```suggestion bold for UI elements
The **Set font size** slider adjusts the **Shell** displayed text size.
```

File: content/SCALE/SystemSettings/ScaleShell.md [hunk 3]
```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
...
+Click *RESTORE DEFAULT* to reset the **Shell** font size.
```
Comment (micjohnson777) [inline content/SCALE/SystemSettings/ScaleShell.md:17]:
```suggestion
Click **RESTORE DEFAULT** to reset the **Shell** font size.
```

File: content/SCALE/SystemSettings/ScaleShell.md [hunk 4]
```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
...
 
-Shell command history is available for the current session.
+The **Shell** stores the command history for the current session.
 Use the <kbd>Up</kbd> and <kbd>Down</kbd> arrow keys to scroll through previously entered commands.
```
Comment (micjohnson777) [inline content/SCALE/SystemSettings/ScaleShell.md:20]:
```suggestion use the icons for the keyboard arrows, but the words up and down are not on the keyboard
Use the keyboard up and down arrow keys to scroll through previously entered commands.
```

File: content/SCALE/SystemSettings/ScaleShell.md [hunk 5]
```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
...
 
-<kbd>Home</kbd>, <kbd>End</kbd>, and <kbd>Delete</kbd> keys are supported.
-Tab completion is also available.
+The **Shell* supports the <kbd>Home</kbd>, <kbd>End</kbd>, and <kbd>Delete</kbd> keys, as well as <kbd>Tab</kbd> completion.
 Type a few letters and press <kbd>Tab</kbd> to complete a command name or filename in the current directory.
-Right-clicking in the terminal window displays a reminder about using <kbd>Command+c</kbd> and <kbd>Command+v</kbd> or <kbd>Ctrl+Insert</kbd> and <kbd>Shift+Insert</kbd> for copy and paste operations in the shell.
+Right-clicking in the terminal window displays a reminder about using <kbd>Command+c</kbd> and <kbd>Command+v</kbd> or <kbd>Ctrl+Insert</kbd> and <kbd>Shift+Insert</kbd> for copy and paste operations in the **Shell**.
 
 Entering `exit` leaves the session.
 Click *Reconnect* to start a new session.
```
Comment (micjohnson777) [inline content/SCALE/SystemSettings/ScaleShell.md:29]:
```suggestion
Click **Reconnect** to start a new session.
```

File: content/SCALE/SystemSettings/ScaleShell.md [hunk 6]
```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
...
 
 ## Default Shell
 
-Clicking other web interface menus closes the shell session and stops commands running in the shell.
-[zsh](https://www.zsh.org/) is the default shell, but this is changed by editing the *root* user in **Credentials > Local Users** and choosing a different option in the *Shell* drop down. Most Linux command line utilities are available in the shell.
+Clicking other web interface menus closes the **Shell** session and stops commands running in the **Shell**.
```
Comment (micjohnson777) [inline content/SCALE/SystemSettings/ScaleShell.md:33]:
```suggestion - bold for direct UI mentions not general function or feature, so in both cases this is talking about shell as the screen or is it about the feature/function shell? kinda murky so I suggest this:
Clicking other web interface menus closes the shell* session and stops commands running in the **Shell** screen.
```

File: content/SCALE/SystemSettings/ScaleShell.md [hunk 7]
```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
...
+[zsh](https://www.zsh.org/) is the default **Shell**, but you can change this by editing the *root* user in **Credentials > Local Users** and choosing a different option in the *Shell* drop-down. Most Linux command-line utilities are available in the **Shell**.
 
-Tmux provides the ability to detach shell sessions and then reattach to them later.
+Tmux allows you to detach **Shell** sessions and then reattach them later.
```
Comment (micjohnson777) [inline content/SCALE/SystemSettings/ScaleShell.md:36]:
```suggestion
[Tmux](https://github.com/tmux/tmux/wiki/) allows you to detach  sessions in **Shell** and then reattach them later.
```

File: content/SCALE/SystemSettings/ScaleShell.md [hunk 8]
```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
...
 Commands continue to run in a detached session.
 
 ## Experimental CLI
 
-Using the experimental SCALE command line interface (CLI) to directly configure different SCALE features.
+The experimental SCALE command-line interface (CLI) lets you configure SCALE features directly.
```
Comment (micjohnson777) [inline content/SCALE/SystemSettings/ScaleShell.md:41]:
```suggestion (directly modifies configure)
The experimental SCALE command-line interface (CLI) lets you directly configure SCALE features.
```

File: content/SCALE/SystemSettings/ScaleShell.md [hunk 9]
```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
 weight: 60
 ---
 
-The SCALE shell is a convenient means to run command lines tools, configure different system settings, or find log files or debug information.
-The shell opens with the root user logged in.
+The SCALE **Shell** is convenient for running command lines tools, configuring different system settings, or finding log files and debug information.
+The **Shell** opens with the root user logged in.
```
Comment (micjohnson777) [inline content/SCALE/SystemSettings/ScaleShell.md:8]:
```suggestion
The **Shell** screen opens with the root user logged in.
```

File: content/SCALE/SystemSettings/ScaleShell.md (hunk 1 above)
Comment (tonyriv3) [reply content/SCALE/SystemSettings/ScaleShell.md:11]: @micjohnson777 
I know the text isn't optimized, but that's exactly how it looks in the Console Setup Menu, so we should probably leave that one as-is.

File: content/SCALE/SystemSettings/ScaleShell.md (hunk 6 above)
Comment (tonyriv3) [reply content/SCALE/SystemSettings/ScaleShell.md:33]: @micjohnson777 
Is that first `**shell***` supposed to be `**shell?**`

Comment (micjohnson777) [reply content/SCALE/SystemSettings/ScaleShell.md:33]: yes... dang typo.

**Comment by tonyriv3** [comment]
@micjohnson777 
This one is ready to merge!

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**